docs/en/03_configuration.md

```diff
@@ -376,12 +396,12 @@ This can be done on configuration level and it can cover even fields which are n
 
 ### Known issues with non-latin characters
 
-Core SilverStripe modules provide text processing functionality but part of it may not work properly with non-latin characters.
+Core Silverstripe modules provide text processing functionality but part of it may not work properly with non-latin characters.
```

Comment:
```suggestion
Core Silverstripe CMS modules provide text processing functionality but part of it may not work properly with non-latin characters.
```

Reply:
Updated